apache/superset
# chore: Remove FilterBox.less

### SUMMARY
Removes the FilterBox.less file from the FilterBox component.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
No visual changes

### TESTING INSTRUCTIONS
1. Create a FilterBox chart
2. Add filters
3. Make sure no visual changes can be observed
4. Add the FilterBox to a Dashboard
5. Open and set all the filters
6. Make sure no visual changes can be observed

### ADDITIONAL INFORMATION
